src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-1n + 2),
+.stars--3 :nth-child(-1n + 3),
+.stars--4 :nth-child(-1n + 4),
+.stars--5 :nth-child(-1n + 5) {
```

Comment:
The usage of `:nth-child` pseudo-class seems incorrect. The `-1n` part is unnecessary and can be removed. It should be simply `.stars--2 :nth-child(2)`, `.stars--3 :nth-child(3)`, and so on, to target the specific child elements.